dashboard-frontend/src/pages/StudyPage/StudySegmentsTables.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import StudySegmentsTable from './StudySegmentsTable';
+import { round, formatNumber } from '../../helpers/numbers';
+
+const StudySegmentsTables = (props: {
+  showLoader: boolean;
+  studySlug: string;
+}) => {
+  const sharedProps = {
+    ...props,
+    loaderTestId: 'study-segment-table-skeleton',
+  };
+
+  return (
+    <React.Fragment>
```

Comment:
Looking at this now, I think it's probably more useful to have these in some sort of "tab" type control, so that you look at one or the other, but you see more rows by default. I think this is perfect for now, but can you make a new issue to throw in the backlog for something like this?

Reply:
Yes I think it's a good idea to think in ways to improve how we present the current progress of the different Segments of an Study. 👌

 I created an issue for it: https://github.com/vlab-research/vlab/issues/29

I already provided some details in the issue, but to give you some context, I went for 3 tables of 4 columns because this way the information can be consumed fine in any screen size. I tried with more columns but the table broke on mobile screen sizes 😢

Reply:
Couple thoughts on this: 

1. I think we need to support tables with more than 4 columns. I don't see a way around that. 

2. Really, nobody should ever need to look at the details of a table on their mobile. If they're looking on their mobile, it's to get an overview. If they're want to dive into the details of a table, it's because they want to see all the info of a table and this is never going to be pleasant on a phone. 

3. Let's make it not "break", but that doesn't mean it has to be an amazing experience. I have the Google Sheets app on my phone. It sucks looking at a Sheet on my phone. I have to scroll left <> right to see the columns. But it works, it's not broken, it's just inherently not an amazing experience. 

4. Options to solve the problem: some way to select which columns are shown. On a mobile, you should select fewer columns.

But really, I don't want to prioritize viewing a table on a mobile. That seems like a rare edge case to me and this can always be determined empirically when we have usage data.

Reply:
Okay makes sense👌  let's go with this approach of having a single table, if it does not fit in the screen, we allow the user to scroll it horizontally as you pointed.

You're right that users won't probably use the app through mobile, and it's better to optimize for the desktop experience for now.